### ISSUE TITLE
[FEATURE] Supply related classifications in the data dump and as an API endpoint

### ISSUE THREAD

**Issue by CopyCat73** [ror-community/ror-roadmap#209]
I am loading ROR organizations in Pure (CRIS system), and need to create the relevant classification schemes in Pure, e.g. the organization type, country code/name, relationship types, external id types. I need to have them present in Pure before I can load the organizations.

**Describe the solution you'd like**
A separate list of classifications in the datadump, e.g. :

[
    classifications [{"type": [ ...],
    organizations: [{
        "id": "https://ror.org/01sf06y89",
        "name": "Macquarie University",
        "types": [
            "Education"

Plus endpoints in the api to query classifications; http://api.ror.org/v1/classifications/types, http://api.ror.org/v1/classifications/relationship-types

This would benefit IT minded people who want to adopt ROR in CRIS systems for instance.

**Comment by CopyCat73** [comment]
Yes, and also list of relation types (related, child, parent), types of id, link types. Basically everything that's a type :)

**Comment by amandafrench** [comment]
Got it! Makes sense. We'll discuss.

**Comment by amandafrench** [comment]
@CopyCat73 Can you tell us more about what you mean by "a separate list of classifications in the data dump"? Would it be sufficient for your needs if we included a separate schema file with every data dump? 

I'm sure you've seen all the documentation at https://ror.readme.io/docs/ror-data-structure : that has the lists of organization types, relationship types, and external id types. It doesn't list all the country codes, but of course anything in the ISO standard is valid.
